src/hiero_sdk_python/staking_info.py

```
@@ -34,7 +36,9 @@ class StakingInfo:
 
     def __post_init__(self) -> None:
         if self.staked_account_id is not None and self.staked_node_id is not None:
-            raise ValueError("Only one of staked_account_id or staked_node_id can be set.")
+            raise ValueError(
+                "Only one of staked_account_id or staked_node_id can be set."
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)